c/indexer.h

```diff
@@ -50,6 +50,7 @@
 #include <zlib.h>
 
 typedef unsigned char uchar;
+typedef int64_t offset_t;
```

Comment:
Since I didn't know them without googling :)

`off_t` is in [POSIX](https://pubs.opengroup.org/onlinepubs/009696899/basedefs/sys/types.h.html) and [GNU libc](https://www.gnu.org/software/libc/manual/html_node/File-Position-Primitive.html).

int64_t is in [C99](http://port70.net/~nsz/c/c99/n1256.html#7.18.1.1), not exactly though.

Reply:
Yep, the reason to use `int64_t` is that it's certain that its size is 8 bytes for any platform. The size of the `off_t` is not guaranteed though.